.pre-commit-hooks.yaml

```diff
@@ -22,3 +22,13 @@
   pass_filenames: false
   files: ^(.*/)?poetry\.lock$
   args: ["-f", "requirements.txt", "-o", "requirements.txt"]
+
+- id: poetry-install
+  name: poetry-install
+  description: Synchronize the Poetry dependencies with the lock file if out of sync
+  entry: poetry install
+  language: python
+  pass_filenames: false
+  stages: [post-checkout, post-merge]
+  always_run: true
+  args: ["--sync"]
```

Comment:
I still think, in respect to the principle of least surprise, we shouldn't add any default arguments. Users might be confused here two times. One time because when reading "poetry-install" they think of the default behavior of `poetry install`. And a second time, when a project don't want to use `--sync`  they have to overwrite the `args` within the pre-commit config with an empty list. This might look unnecessary to people reading this config.

Reply:
Now I got you and thanks for the patience.